### ISSUE TITLE
Auto-register routes

### ISSUE THREAD

**Issue by zupo** [Pylons/pyramid_openapi3#46]
Would it make sense to have `pyramid_openapi3` automatically register routes for all endpoints defined in`openapi.yaml`? This could be a default behavior so it's easier to get started with and less configuration is needed for simpler APIs.

**Comment by jenner** [comment]
I like the fact that the package isn't opinionated on how you use it. Automatically registered routes would make it opinionated though (IMO). Adding an optional config method like config.pyramid_openapi3_register_routes() might be the way, so the routes are still need to be defined in wsgi factory if it's not called.

**Comment by gweis** [comment]
Yes please, ... keep that feature optional (if it ever get's implemented)

**Comment by slobinger** [comment]
Yes it would be kind of opinionated like everything developed. IMO connexion has made a good decision in *how* they automatically map routes defined in spec to view functions.
OpenAPI has the 'operationID' that is perfect for this as one could write kind of 'fully quallified function name' e.g. package.(subpackages).module.(object).method in it so that 
pyramid_openapi3  does not need to make a decision for the developer.

**Comment by zupo** [comment]
Initial implementation released in https://pypi.org/project/pyramid-openapi3/0.12/.

**Comment by miketheman** [comment]
As above, initial implementation was released in 0.12, and extended in #204.

Considering that the feature remains opt-in, and no further requests surfaced to enable this as the default behavior, closing.
